doc/nrf/working_with_nrf/nrf91/nrf9160.rst

```diff
@@ -30,6 +30,10 @@ Build the sample (located under ``ncs/zephyr/samples/hello_world``) for the nrf9
 To change the routing options, enable or disable the corresponding devicetree nodes for that board as needed.
 See :ref:`zephyr:nrf9160dk_board_controller_firmware` for detailed information.
 
+To use the external flash on the nRF9160 DK, Program the board controller firmware (:file:`nrf9160_dk_board_controller_fw_2.0.1.hex`) v2.0.1 or higher using the `Programmer app <Programming a Development Kit_>`_ in nRF Connect for Desktop.
+Make sure the **PROG/DEBUG SW10** switch on the nRF9160 DK is set to **nRF52**.
+On DK v0.9.0 and earlier, this is the **SW5** switch.
```

Comment:
The v0.9.0 board and earlier didn't have external flash, so this line can be removed.